### PR TITLE
Implemented RSA Cipher

### LINKED ISSUE #9: Chiffrement
En tant que développeur, je veux que les données soient chiffrées afin de garantir la sécurité des données des utilisateurs.

Valeur: 1

### PR DESCRIPTION
**Pull request by PierreCorentin**
Closes #9

### CODE REVIEW

File: ORA.Core/Encryption/RSACipher.cs [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+using System.Security.Cryptography;
+
+namespace ORA.Core.Encryption
+{
+    public class RsaCipher : ICipher
+    {
+        private RSAParameters publickey;
+        private RSAParameters privatekey;
+
+        public RsaCipher()
```
Comment (adamaq01) [inline ORA.Core/Encryption/RSACipher.cs]: Could be a great idea to let the choice of the size of the keys in the constructor

Comment (adamaq01) [reply ORA.Core/Encryption/RSACipher.cs]: @TrAyZeN ?

**Review by adamaq01 — CHANGES_REQUESTED** [review]
As commented on the code, it could be a great idea to let the choice of the keys size in the constructor of RsaCipher and to store it.

File: ORA.Core/Encryption/RSACipher.cs (hunk 1 above)
Comment (TrAyZeN) [reply ORA.Core/Encryption/RSACipher.cs]: We will be using only one size of key, but if it makes sense to you that people who want to build something on top of ORA want to change the key size then yes this is a good idea.

**Review by adamaq01 — APPROVED** [review]

**Pull request merged**